### PR TITLE
Bug 1419761 - Rename "histogram_tools.py" to "parse_histograms.py"

### PR DESCRIPTION
**Pull request by RyanLeake**
Rename histogram_tools and update_histogram_tools
Replace instances of "histogram_tools" with "parse_histograms"
Fixes bug 1419761

### CODE REVIEW

**Comment by Dexterp37** [comment]
This looks good to me but I don't own this repo :) So .. r? @mreid-moz  probably :-D

File: README.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ A new pypi release will be automatically triggered by Travis.
 pip install python_moztelemetry
 ```
 
-## Updating histogram_tools.py
-moztelemetry/histogram_tools.py is a mirror of its counterpart from
-[mozilla-central](https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/telemetry/histogram_tools.py).
+## Updating parse_histograms.py
```
Comment (mreid-moz) [inline README.md]: Please add a quick note to this section to help future maintainers saying something like:

Note that this file was formerly called `histogram_tools.py` and was renamed in [Bug 1419761](https://bugzilla.mozilla.org/show_bug.cgi?id=1419761).

File: bin/update_parse_histograms [hunk 2]
```diff
@@ -7,22 +7,22 @@ import os
 import requests
 
 
-"""Replace histogram_tools.py with the latest version from mozilla-central"""
+"""Replace parse_histograms.py with the latest version from mozilla-central"""
```
Comment (mreid-moz) [inline bin/update_parse_histograms]: > bin/update_histogram_tools → bin/update_parse_histograms 100755 → 100644

This file should retain it's executable permissions.

**Comment by RyanLeake** [comment]
@mreid-moz I think that should be good to go?

**Pull request merged**